python/chronos/test/bigdl/chronos/data/test_tsdataset.py

```diff
@@ -1344,3 +1344,75 @@ def test_tsdataset_interval_check_and_repair_for_multi_id(self):
                                         lookback=lookback,
                                         horizon=horizon,
                                         target_col=['value', 'extra feature'])
+
+    def test_from_prometheus(self):
+        '''
+        The selected query str is just for getting data from Prometheus server.
+        '''
+        import time
+        endtime = time.time()
+        starttime = endtime - 1000
+        tsdata = TSDataset.from_prometheus(prometheus_url='http://10.112.231.51:19090/',
```

Comment:
we may avoid setting the url explicitly, alternatively we may use `os.getenv("PROMETHEUS_URL")` and set this parameter for all action runners like 
https://github.com/intel-analytics/BigDL/blob/c10b4ceee260dd011acfd038328008d048b7623e/python/chronos/test/bigdl/chronos/simulator/test_doppelganger_simulator.py#L58

Reply:
done